README.md

```diff
@@ -1,9 +1,108 @@
 # edgex-checkbox-provider
 
-This project contains the Checkbox tests of the [EdgeX Foundry](https://docs.edgexfoundry.org/) snaps.
+[![GitHub Latest Dev Tag)](https://img.shields.io/github/v/tag/canonical/edgex-checkbox-provider?include_prereleases&sort=semver&label=latest-dev)](https://github.com/canonical/edgex-checkbox-provider/tags) 
+[![GitHub License](https://img.shields.io/github/license/canoical/edgex-checkbox-provider)](https://choosealicense.com/licenses/apache-2.0/) 
+[![GitHub Pull Requests](https://img.shields.io/github/issues-pr-raw/canonical/edgex-checkbox-provider)](https://github.com/edgexfoundry/edgex-cli/pulls) 
+[![GitHub Contributors](https://img.shields.io/github/contributors/canonical/edgex-checkbox-provider)](https://github.com/edgexfoundry/edgex-cli/contributors) 
+[![GitHub Committers](https://img.shields.io/badge/team-committers-green)](https://github.com/orgs/canonical/teams/edgex/members) 
+[![GitHub Commit Activity](https://img.shields.io/github/commit-activity/m/edgexfoundry/edgex-cli)](https://github.com/canonical/edgex-checkbox-provider/commits)
```

Comment:
I think many of the badges are redundant. All of tag, license, pull req, contributors, committers, commit activity are already available on github.

Reply:
Removed.